pkg/ruler/compat.go

```diff
@@ -199,12 +206,20 @@ func DefaultTenantManagerFactory(cfg Config, p Pusher, q storage.Queryable, engi
 		Name: "cortex_ruler_queries_failed_total",
 		Help: "Number of failed queries by ruler.",
 	})
+	var rulerQuerySeconds *prometheus.CounterVec
+	if cfg.RulerEnableQueryStats {
+		rulerQuerySeconds = promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Name: "cortex_ruler_query_seconds_total",
+			Help: "Total amount of wall clock time spend processing queries by the ruler.",
```

Comment:
Could you also fix the typo for the `cortex_query_seconds_total` metric, please?

```suggestion
			Help: "Total amount of wall clock time spent processing queries by the ruler.",
```